aries-site/src/examples/cardPreviews/textinput 2.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { TextInput } from 'grommet';
+
+export const TextInputPreview = () => {
+  return <TextInput focusIndicator placeholder="Placeholder text" />;
```

Comment:
why do we need an external component that only performs one line without a logic? and why does the file called `textinput 2.js`?
why number two? isn't it a component and it should be using upper case?

Reply:
the same comment for all the files that use the number two.